elasticsearch/README.md

```diff
@@ -16,7 +16,6 @@ This helm chart is a lightweight way to configure and run our official [Elastics
 
 * This repo includes a number of [example](./examples) configurations which can be used as a reference. They are also used in the automated testing of this chart
 * Automated testing of this chart is currently only run against GKE (Google Kubernetes Engine). If you are using a different Kubernetes provider you will likely need to adjust the `storageClassName` in the `volumeClaimTemplate`
```

Comment:
The comment below is still totally valid. This is the line that is wrong now that the default storage class is undefined.  
```suggestion
* Automated testing of this chart is currently only run against GKE (Google Kubernetes Engine).
```